core/src/clients/git.rs

```diff
@@ -407,7 +407,8 @@ impl Git {
             stash_args.push(path);
         }
 
-        self.run(&stash_args, Opts::default()).await?;
+        self.run(&stash_args, Opts::new_with_ignored(&["unable to unlink"]))
```

Comment:
I actually do not know why we can get away with this.

Reply:
this is good enough for me, ty